hack/tools/logcheck/pkg/logcheck.go

```diff
@@ -254,6 +263,30 @@ func isUnstructured(fName string) bool {
 	return false
 }
 
+func isLegacyKlogCall(fName string) bool {
```

Comment:
I would refrain from using word "legacy" as it's meaning can drift over time. Before contextual logging, legacy was just fstring methods (klog.Infof), now you are using it for non-structured calls (klog.InfoS). 

Even the naming of slice in function suggest that different name of function. I think we should prefer "isContextualCall" instead.

Reply:
Okay.